conf/binac.config

```diff
@@ -17,7 +16,8 @@ process {
 }
 
 params {
-  igenomesIgnore = true
+  igenomesIgnore = false
```

Comment:
I think this is the default so can be removed in the future if you want...

Reply:
True, can drop it as well :-)